### PR TITLE
Update Submariner OVN Documentation

### LINKED ISSUE #779: Update OVN support documentation
Update OVN support docuentation with changes made for 0.13.0 release.

- https://github.com/submariner-io/submariner-website/pull/699

### PR DESCRIPTION
**Pull request by astoycos**
This commit updates the documents involving the
submariner + OVN-K integration, and also re-does
the architecture diagram.

Signed-off-by: Andrew Stoycos <astoycos@redhat.com>

### CODE REVIEW

**Comment by astoycos** [comment]
/hold 

until OVN related submariner PR's have merged

**Comment by nyechiel** [comment]
@astoycos as you probably noticed, we don't use Prow in this org so you have to manually add labels, etc. To mark a PR as draft, we typically use https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request as documented [here](https://submariner.io/development/code-review/#mark-work-in-progress-prs-as-drafts).

**Comment by astoycos** [comment]
Yep no worries!

**Review by dfarrell07 — APPROVED** [review]
This looks outstanding, major fixes/improvements. Thank you @astoycos!

Are we ready to undraft/review/merge?

File: src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md
Comment (sridhargaddam) [inline src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md:43]: Nice diagram with explanation @astoycos, thanks.

**Comment by astoycos** [comment]
@dfarrell07 This should be pretty much ready to go, I just wonder if we should wait until https://github.com/submariner-io/submariner/pull/1689? Although there isn't any major architectural changes, I may need to edit/augment these docs based on reviews

File: src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md
Comment (sridhargaddam) [reply src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md:43]: @astoycos In the attached diagram, On OVN Submariner router, it is mentioned that "OVN Nat rules are applied here to ensure traffic is SNAT'ed to the globalnetIP when Globalnet is enabled".

Since this will be done when we support OVN with Globalnet and is not there currently, I'm not sure if we should capture this in the diagram. Basically, it might give an impression that Globalnet is supported even with OVN by looking into the diagram. What you do say?

**Comment by nyechiel** [comment]
@vthapar this one should be ready to get merged now, can you please take it?

File: src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md
Comment (sridhargaddam) [inline src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md]: Can you please copy the original image to the following PPT [*] and add the following comment which will help us to make further changes in future if required.
```
<!-- Image Source: https://docs.google.com/presentation/d/180CtHZnr9PP5Rh98VEmkQz3ovc5AGXG9wosoHMLhgaY/edit -->

```

[*] https://docs.google.com/presentation/d/180CtHZnr9PP5Rh98VEmkQz3ovc5AGXG9wosoHMLhgaY/edit#slide=id.g8e581e9edd_0_142

File: src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md
Comment (astoycos) [reply src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md:43]: Done

File: src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md
Comment (astoycos) [reply src/content/getting-started/architecture/networkplugin-syncer/ovn-kubernetes/_index.en.md]: Done

**Review by nyechiel — APPROVED** [review]
Thanks @astoycos

**Comment by sridhargaddam** [comment]
Thanks @astoycos

**Pull request merged**
